distmv/statdist.go

```diff
@@ -12,6 +12,14 @@ import (
 	"github.com/gonum/stat"
 )
 
+type DistNormaler interface {
```

Comment:
I don't think these are justifiable here. Can you not define them in your package?

(Also, no docs).